pkg/api/customization/alert/notifier_action.go

```diff
@@ -100,11 +100,18 @@ func (h *Handler) testNotifier(actionName string, action *types.Action, apiConte
 			host, ok := smtpConfig["host"].(string)
 			if ok {
 				port, _ := smtpConfig["port"].(json.Number).Int64()
-				password := smtpConfig["password"].(string)
+				password, ok := smtpConfig["password"].(string)
```

Comment:
I'd suggest you use `convert` package to do strings conversion. Its string() method would do automatic defaulting to empty string as well, so the line 104 is no longer needed. Same for username and the sender

Reply:
Sure